backend/lobby/consumers.py

```diff
@@ -79,14 +81,20 @@ def get_or_create_user(self, username, room):
 
     def get_or_create_game(self, username, room):
         game, created = GameState.objects.get_or_create(room_name=room)
-
         i_repository_game = IRepositoryGame()
+        i_repository_player = IRepositoryPlayer()
 
         if not i_repository_game.get_game_by_room(room):
             game_store = i_repository_game.save_game(room)
-            print("Room created with id {}".format(game_store))
+            print("Room created in analytics database with id {}".format(game_store))
+        else:
+            error = 'Room already exist in analytics database, Unable to create Game with room: ' + room
+
+        if not i_repository_player.get_players_by_username_and_room(username, room):
+            player_store = i_repository_player.save_player(username, room)
+            print("Player created in analytics database with id {}".format(player_store))
         else:
-            error = 'Room already exist, Unable to create Game with room: ' + room
+            error = 'Player in this room already exist in analytics database, Unable to create Player in room: ' + room
```

Comment:
is this really an error? If the room exists, were just getting it right?

second message, if players in room... were just utilizing past state?

this is through, but maybe not necessary?

Since the messages are passive, I'm approving.